examples/features/mutually-awared-nses/README.md

```diff
@@ -0,0 +1,178 @@
+# Test kernel to kernel connection
+
+
+This example shows that NSC and NSE on the one node can find each other. 
+
+NSC and NSE are using the `kernel` mechanism to connect to its local forwarder.
+
+## Requires
+
+Make sure that you have completed steps from [basic](../../basic) or [memory](../../memory) setup.
+
+## Run
+
+Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/ff873ab1eec320c2abf6ba380d95137e71a969b2/examples/use-cases/namespace.yaml)[0])
+NAMESPACE=${NAMESPACE:10}
+```
+
+Select node to deploy NSC and NSE:
+```bash
+NODE=($(kubectl get nodes -o go-template='{{range .items}}{{ if not .spec.taints  }}{{index .metadata.labels "kubernetes.io/hostname"}} {{end}}{{end}}')[0])
+```
+
+Create customization file:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ${NAMESPACE}
+
+resources:
+- nse-1.yaml
+- nse-2.yaml
+- config-file-nse-1.yaml
+- config-file-nse-2.yaml
+bases:
+- https://github.com/networkservicemesh/deployments-k8s/apps/nsc-kernel?ref=ff873ab1eec320c2abf6ba380d95137e71a969b2
+
+patchesStrategicMerge:
+- patch-nse-1.yaml
+- patch-nse-2.yaml
+- patch-nsc.yaml
+EOF
+```
+
+Create Client:
+```bash
+cat > patch-nsc.yaml <<EOF
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nsc-kernel
+spec:
+  template:
+    spec:
+      containers:
+        - name: nsc
+          env:
+            - name: NSM_NETWORK_SERVICES
+              value: kernel://icmp-responder-1/nsm-1?color=red,kernel://icmp-responder-2/nsm-2?color=red
+            - name: NSM_AWARENESS_GROUPS
+              value: "[kernel://icmp-responder-1/nsm-1?color=red,kernel://icmp-responder-2/nsm-2?color=red]"
+      nodeSelector:
+        kubernetes.io/hostname: ${NODE}
+EOF
+```
+
+Create NSE-1 patch:
+```bash
+cat > patch-nse-1.yaml <<EOF
```

Comment:
This is a patch for local file `nse-1.yaml` and the only dynamic part in the patch is `nodeSelector`. Can we move non-dynamic props (`env`, `volumes`, etc) into `nse-1.yaml`? This will reduce the length of this readme.